docs/user-guide/getting-started.md

```diff
@@ -6,6 +6,13 @@
 * Set the `JAVA_HOME` variable to point to your Java installation.
 * Set the `PATH` variable to include `$JAVA_HOME/bin`.
 
+## System requirements
+
+In order to build Strongbox in reasonably short times you need:
+* At least 16 GB RAM
+* An Intel i7 processor (or equivalent)
+* An SSD
```

Comment:
The specs you've listed are for `building` Strongbox from sources which should be located in `docs/developer-guide/getting-started.md`. The ones below is approximately what you'd need to `run` an instance as a user (i.e. through the released `strongbox-distribution`).

```suggestion
In order to run Strongbox you need to have a machine with minimum:

* 6-8 GB RAM
* 2 cores
* Some free space.
```
(note the empty line which fixes the bullet points)